src/CarbonAware.DataSources/CarbonAware.DataSources.WattTime/src/Client/WattTimeClient.cs

```diff
@@ -238,7 +249,8 @@ private async Task UpdateAuthTokenAsync()
     private void SetBasicAuthenticationHeader()
     {
         var authToken = Encoding.UTF8.GetBytes($"{this._configuration.Username}:{this._configuration.Password}");
-        this._client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue(AuthenticationHeaderTypes.Basic, Convert.ToBase64String(authToken));
+        //this._client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue(AuthenticationHeaderTypes.Basic, Convert.ToBase64String(authToken));
```

Comment:
We can drop the comment? (`this._client` will not exist after the PR comes through in this context)